pkg/storaged/content-views.jsx

```diff
@@ -224,7 +224,7 @@ function create_tabs(client, target, is_partition, is_extended) {
     } else if (content_block && (content_block.IdUsage == "raid" ||
                                  client.legacy_vdo_overlay.find_by_backing_block(content_block))) {
         // no tab for these
-    } else if (content_block && content_block.IdUsage == "other" && content_block.IdType == "swap") {
+    } else if (block_swap || (content_block && content_block.IdUsage == "other" && content_block.IdType == "swap")) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17929-20221117-190140-3f947163-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)